src/components/TasksList/TabPanel.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Box from '@material-ui/core/Box';
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: any;
+  value: any;
+}
+
+function TabPanel(props: TabPanelProps) {
```

Comment:
It's working, but I guess we agreed to use const for component definition and setup types for React component
Should be more like this:
```
const TabPanel: React.FC<TabPanelProps> = (props: TabPanelProps) => {
  const { children, value, index, ...other } = props;

  return (
    <Box hidden={value !== index} {...other} p={3}>
      {children}
    </Box>
  );
};
```

Reply:
fixed